libs/community/langchain_community/vectorstores/databricks_vector_search.py

```diff
@@ -321,6 +324,134 @@ def similarity_search_with_score(
         )
         return self._parse_search_response(search_resp)
 
+    def _select_relevance_score_fn(self) -> Callable[[float], float]:
+        """
+        The 'correct' relevance function
+        may differ depending on a few things, including:
+        - the distance / similarity metric used by the VectorStore
+        - the scale of your embeddings (OpenAI's are unit normed. Many others are not!)
+        - embedding dimensionality
+        - etc.
+        """
+
+        def _db_relevance_score_fn(distance: float) -> float:
+            """
+            Scores returned by the Databricks Vector Search API are
+            normalized cosine similarity scores on a scale [0, 1] -
+            1 means most similar
```

Comment:
Can we confirm the similarity metric used by Databricks Vector Search?

Reply:
This has now been confirmed to be a normalized score 1/(1+d) with d being the L2-distance. I've added that information to the code.